nav2_behavior_tree/include/nav2_behavior_tree/bt_action_server_impl.hpp

```diff
@@ -67,16 +68,22 @@ BtActionServer<ActionT>::BtActionServer(
   };
 
   if (!node->has_parameter("error_code_names")) {
```

Comment:
This has parameter will always be false if its not been declared, so it can be removed.

Reply:
This needs to be here when the parameter in defined in the yaml. Otherwise there is a declaration error. 

`[component_container_isolated-5] [ERROR] [1687029994.187670094] []: Original error: parameter 'error_code_names' has already been declared`